packages/react/src/ErrorBoundary.tsx

```diff
@@ -71,7 +69,14 @@ export type ErrorBoundaryProps = PropsWithDevMode<
      */
     shouldCatch?: ShouldCatch | [ShouldCatch, ...ShouldCatch[]]
   }>,
-  ErrorBoundaryDevModeProp
+  {
+    showFallback?:
+      | boolean
+      | {
+          errorMessage?: string
+          after?: number
+        }
+  }
```

Comment:
Is showFallback also going to be deprecated? If so, it would be great to note that in the JSDoc.

Reply:
Thanks! I added you as co-author